components/common/IdentityForm.vue

```diff
@@ -122,13 +124,14 @@ const identity = ref<Record<string, string>>({
   legal: '',
 })
 const deposit = ref('0')
-const inputLengthLimit = ref(32)
+const inputLengthLimit = ref('32')
 
 onBeforeMount(async () => {
   onApiConnect(apiUrl.value, async (api) => {
     deposit.value = api.consts.identity?.basicDeposit?.toString()
     identity.value = await fetchIdentity(accountId.value)
   })
+  useIdentityStore().fetchBalance({ address: accountId.value })
```

Comment:
fix bug where amount was not always shown in identity form
needed to open wallet modal to make it fetch balance

Reply:
maybe check getters if empty before invoke `fetchBalance`

Reply:
done